CMake/FindFFMPEG.cmake

```diff
@@ -0,0 +1,89 @@
+# Find the FFmpeg library
+#
+# Sets
+#   FFMPEG_FOUND.  If false, don't try to use ffmpeg
+#   FFMPEG_FOUND_SEVERAL. If true, there are several directories with headers (not only ../ffmpeg/)
+#   FFMPEG_INCLUDE_DIR
+#   FFMPEG_LIBRARIES
+
+set( FFMPEG_FOUND "NO" )
+
+find_path( FFMPEG_INCLUDE1_DIR ffmpeg/avcodec.h
```

Comment:
Since this is kwiver code, should we add some priming of paths from fletch_DIR? We probably want to give preference to that package to ensure they continue to work together.